avs_client/avs_client/connection.py

```diff
@@ -60,18 +60,16 @@ def synchronise_device_state(self, device_state, authentication_headers):
             ],
             boundary='boundary'
         )
-        headers = {
-            **authentication_headers,
-            'Content-Type': multipart_data.content_type
-        }
+        headers = {'Content-Type': multipart_data.content_type}
+        headers.update(**authentication_headers)
```

Comment:
this change is undesirable.

## Before the change ##
`'Content-Type'` was guaranteed to be the value of `multipart_data.content_type`.

## After the change ##
`'Content-Type'` can be overwritten by value inside `authentication_headers`.

Please change this back to
```
headers = {
    **authentication_headers,
    'Content-Type': multipart_data.content_type
}
```